host-interaction/process/terminate/terminate-process-via-fastfail.yml

```diff
@@ -1,17 +0,0 @@
-rule:
```

Comment:
ref: #143, I've only seen this hit in library code (which Python flirt often doesn't recognize)